src/Fractions/Fraction.ConvertFrom.cs

```diff
@@ -88,26 +89,36 @@ namespace Fractions;
     /// <para><c>true</c> if <paramref name="fractionString"/> was well-formed. The parsing result will be written to <paramref name="fraction"/>. </para>
     /// <para><c>false</c> if <paramref name="fractionString"/> was invalid.</para>
     /// </returns>
-    public static bool TryParse(string fractionString, NumberStyles numberStyles, IFormatProvider formatProvider,
+    public static bool TryParse(ReadOnlySpan<char> fractionString, NumberStyles numberStyles, IFormatProvider formatProvider,

```

Comment:
Please note that I removed the string version overload in .net8 and renamed the parameter name to match the other one. 
Technically this is a breaking change, but only for people who use reflection (or who have recently switched to using the ReadOnlySpan overload _and_ have named the parameter explicitly).

Personally, I think this is very unlikely to break for anyone, while the advantage (IMO) of not having the `string` overload is that the user would probably notice the little icon indicating an implicit conversion, and consider updating his code with ReadOnlySpan<char>.. 🚀

Reply:
I don't think changing the public API really brings much benefit here. [Named Arguments](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments) are used quite frequently (at least by us).

I'll add the overload again for the git-merge.